llvm/lib/Transforms/Vectorize/SandboxVectorizer/Passes/PassRegistry.def

```diff
@@ -14,9 +14,19 @@
 // NOTE: NO INCLUDE GUARD DESIRED!
 
 #ifndef REGION_PASS
-#define REGION_PASS(NAME, CREATE_PASS)
+#define REGION_PASS(NAME, CLASS_NAME)
 #endif
 
-REGION_PASS("null", NullPass())
+REGION_PASS("null", ::llvm::sandboxir::NullPass)
```

Comment:
We don't NEED to, but with the vectorizer having code in both the `llvm` and `llvm::sandboxir` namespaces, I thought this makes it easier to include the .def file without having to care about the surrounding namespace.

Now that I've introduced the SandboxVectorizerPassBuilder helper it's less of a concern because that's probably the only place that will need to include the .def file. I think it doesn't hurt, but if you'd prefer not having the qualifier I can remove it.

Reply:
I don't feel strongly about it, but if it works fine without the qualifiers I would prefer them that way because it's just easier to read.